chain/chain/src/flat_storage_resharder.rs

```diff
@@ -357,8 +364,13 @@ impl FlatStorageResharder {
         let (parent_shard, split_status) = self
             .get_parent_shard_and_status()
             .expect("flat storage resharding event must be Split!");
-        let SplittingParentStatus { left_child_shard, right_child_shard, flat_head, .. } =
-            split_status;
+        let SplittingParentStatus {
+            left_child_shard,
+            right_child_shard,
+            flat_head,
+            block_hash,
```

Comment:
nit: What is this block hash? Can you rename it to something more telling? Maybe something similar to sync hash in state sync like `resharding_hash` that is well defined?

Reply:
Done!